src/main/java/com/glencoesoftware/bioformats2raw/MiraxReader.java

```diff
@@ -1069,10 +1069,17 @@ public void setFillValue(Byte fill) {
    * for brightfield.
    *
    * @return fill value for missing tiles
+   * @deprecated use getFillColor()
```

Comment:
As this is planned for `0.10.0`, is there a value in going through a deprecation rather than dropping these in favor of the new API defined upstream?

Reply:
I wasn't totally sure what the best approach would be here, so went with deprecating to be on the safer side. If we're all OK with just removing `getFillValue()` and `setFillValue()`, I'm fine with that - it would only impact anyone using `MiraxReader` directly, either by extending the reader or by calling those methods directly.